javascript/javascript1/week3/arrival-time.js

```diff
@@ -0,0 +1,19 @@
+const travelInformation = {
+    speed: 80,
+    destinationDistance: 573,
+};
+
+function findTime(information) {
+    return information.destinationDistance / information.speed; //Basic physics time calculation formula: Time = distance / speed
+}
+
+const travelTime = findTime(travelInformation);
+let hours = Math.floor(travelTime);
+let minutes = Math.ceil((travelTime % Math.floor(travelTime)) * 100);
```

Comment:
This should be 60 not 100
`Math.floor(travelTime)` is hours, so you can replaced